magica/template/quest/scene0/Top.html

```diff
@@ -0,0 +1,19 @@
+<div id="Scene0Top" class="<%= model.templateClass %> <%= model.terminalClass %>">
+	<div id="mainSec" class="mainSec">
+		<div id="mainStoryBtn" class="mainStoryBtn se_decide TE"></div>
+		<div id="battleBtn" class="battleBtn se_decide TE"></div>
+		<div id="sideStoryBtn" class="sideStoryBtn se_decide TE"></div>
+		<div id="missionBtn" class="missionBtn se_decide TE">
+			<div class="balloonSec <%= model.limitedMission.balloonSecClass %>">
+				<div class="textSec">
+					<span class="textDate"><%= model.limitedMission.endDateText %><span>Until</span></span><br>
```

Comment:
I assume it should be reordered this way?
```suggestion
					<span class="textDate"><span>Until </span><%= model.limitedMission.endDateText %></span><br>
```